### ISSUE TITLE
about pre-embed file

### ISSUE THREAD

**Issue by Marymarymaryjob** [SixingPKU/ACL2020-ConKADI#5]
When training the model, all the vectors are randomly initialized. The pre-trained language model was used in the evaluation. Is that true?

**Comment by pku-sixing** [comment]
Yep! Vectors are randomly initialized in training. And we use a pre-trained embedding to evaluate.

**Comment by Marymarymaryjob** [comment]
Okay, one more question is why don't you use a pre-trained language model to initialize all the vectors when you train the model

**Comment by Marymarymaryjob** [comment]
> Yep! Vectors are randomly initialized in training. And we use a pre-trained embedding to evaluate.

Okay, one more question is why don't we use a pre-trained language model when we train the model?

**Comment by Marymarymaryjob** [comment]
> Yep! Vectors are randomly initialized in training. And we use a pre-trained embedding to evaluate.

Okay, one more question is why don't you use a pre-trained language model to initialize all the vectors when you train the model?

**Comment by pku-sixing** [comment]
Because we find the result is good enough, so we do not try this. 

Meanwhile, some baselines do not use the pre-trained embedding , so we keep the same setting.

**Comment by Marymarymaryjob** [comment]
> Because we find the result is good enough, so we do not try this.
> 
> Meanwhile, some baselines do not use the pre-trained embedding , so we keep the same setting.

OK, thank you for your patient reply.

**Comment by pku-sixing** [comment]
haha~
